hs_model_program/static/css/model_program.css

```diff
@@ -0,0 +1,56 @@
+
+.mpElement{
+    font-weight: bold;
+    padding: 2px 2px 2px 2px;
+}
+
+a.mpElement::after {
+    display:block;
+    /*content:attr(title);*/
+    font-weight:bold;
+    height:2px;
+    color:transparent;
+    overflow:hidden;
+    visibility:hidden;
+}
+
+div#mp.row {
+    padding: 5px 0px 5px 0px
```

Comment:
Can be rewritten as 'padding: 5px 0;'. Also, if the value is 0 you don't need to specify the unit. In this case '0px' would be '0'